source/iostreams.tex

```diff
@@ -7965,7 +7965,7 @@
 Constructs an object of class
 \tcode{basic_istringstream<charT, traits>},
 initializing the base class with
-\tcode{basic_istream<charT, traits>(\&sb)}\iref{istream}
+\tcode{basic_istream<charT, traits>(addressof(sb))}\iref{istream}
 and initializing \tcode{sb} with
 \tcode{basic_string\-buf<charT, traits, Allocator>(str, which | ios_base::in)}\iref{stringbuf.cons}.
 \end{itemdescr}
```

Comment:
This is missing the `basic_istream<charT, traits>::set_rdbuf(\&sb)`on line 7983 below.